### ISSUE TITLE
Quick jump options for Livewire components

### ISSUE THREAD

**Issue by jdjebi** [codingyu/laravel-goto-view#56]
Hi, I don't have the knowledge to contribute to a vscode extension but I think it would be cool to be able to make a quick jump to the syntaxes integrating the livewire components in a view like the ones below:

`<livewire:my-component :data="..." :wire:key="...">` and `@livewire('my-component',[...],key(...))`

**Comment by bilogic** [comment]
I agree!! Actually, make it more generic: currently I'm writing `{{-- @include("Livewire.Example") --}}` and it jumps to my component code.

It would be great if we can have something non-blade such as `@codejump("Livewire.Example")` so that it breaks if not wrapped in comments, that makes it clear it is just a hyperlink.
